webapp/channels/src/components/profile_popover/profile_popover_controller.tsx

```diff
@@ -126,6 +134,10 @@ export function ProfilePopoverController<TriggerComponentType = HTMLSpanElement>
                                 ref={refs.setFloating}
                                 style={{...floatingStyles, ...transitionStyles}}
                                 className={classNames('user-profile-popover', A11yClassNames.POPUP)}
+                                aria-label={intl.formatMessage(
+                                    {id: 'profile_popover.aria_label', defaultMessage: '{name}profile popover'},
+                                    {name: props.username ? `${props.username}'s ` : ''},
```

Comment:
The `'s` needs to be part of the translation as well since it'll vary by language. I think we should just go for two translation strings here: one when we have `username` and another when we don't